src/geometry/manifold/sheaf/smooth.lean

```diff
@@ -0,0 +1,229 @@
+/-
+Copyright © 2023 Heather Macbeth. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Heather Macbeth
+-/
+import geometry.manifold.sheaf.basic
+import geometry.manifold.algebra.smooth_functions
+import category_theory.sites.whiskering
+
+/-! # The sheaf of smooth functions on a manifold -/
```

Comment:
Can you expand the file-level docstring, and add a warning about the current universe issue?